event_webhook_types_test.go

```diff
@@ -185,24 +184,24 @@ func TestMergeEventUnmarshal(t *testing.T) {
 	if event.ObjectAttributes.LastCommit.ID != "da1560886d4f094c3e6c9ef40349f7d38b5d27d7" {
 		t.Errorf("ObjectAttributes.LastCommit.ID is %v, want %s", event.ObjectAttributes.LastCommit.ID, "da1560886d4f094c3e6c9ef40349f7d38b5d27d7")
 	}
-	if event.ObjectAttributes.Assignee.Name != expectedName {
-		t.Errorf("Assignee.Name is %v, want %v", event.ObjectAttributes.Assignee.Name, expectedName)
+	if event.Assignees[0].Name != expectedName {
+		t.Errorf("Assignee.Name is %v, want %v", event.Assignees[0].Name, expectedName)
```

Comment:
Instead of changing the existing tests, please add new ones if you want to add more tests. So please make sure all the existing tests aren't changed or removed.

Reply:
I altered the test, as the payload doesn't include an assignee name in the object attributes (only an id). Should I add fields to the payload, which deviate from the (current) documentation, instead?